backend/tests/unit/scrape/agencyType.test.js

```diff
@@ -0,0 +1,34 @@
+const each = require('jest-each').default;
+
```

Comment:
If you're having the same tests for different submodules maybe an abstraction should emerge? Will those tests still pass after resolving the empty element issue?